versioned_docs/version-1.5.0/install-kubeslice/yaml/yaml-controller-install.mdx

```diff
@@ -0,0 +1,277 @@
+- The KubeSlice Controller workload
+- The KubeSlice Controller API Gateway
+
+## Create a KubeSlice Controller YAML
+
+To install the KubeSlice Controller on one of the clusters, you need to create a `controller.yaml` file that requires the 
+endpoint of the controller cluster. The endpoint is the location on which you install the KubeSlice Controller.
+
+### Get the Cluster Endpoint
+
+Use the following command to get the cluster endpoint:
+
+```
+kubectl cluster-info
+```
+
+Example output
+
+```
+Kubernetes control plane is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443
+addon-http-application-routing-default-http-backend is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/addon-http-application-routing-default-http-backend/proxy
+addon-http-application-routing-nginx-ingress is running at http://40.125.122.238:80 http://40.125.122.238:443
+healthmodel-replicaset-service is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/healthmodel-replicaset-service/proxy
+CoreDNS is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443/ap
+```
+
+From the above output, copy the URL for the **Kubernetes control plane** to
+add it as the cluster endpoint in the `controller.yaml` file.
+
+For example,
+`https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443`.
+
+### Controller Configuration Parameters
+The following tables describe the configuration parameters used to install the KubeSlice Controller.
+
+| Parameter                                                    | Parameter Type | Description                                                                  | Required  |
+| ------------------------------------------------------------ | -------------- | ---------------------------------------------------------------------------- | --------- |
+| Kubeslice                                                    | Object         | This contains the controller and the namespace configuration parameters.     | Mandatory |
+
+#### KubeSlice Parameters
+
+This parameter contains the configuration object used in the
+[KubeSlice Controller YAML file](#create-a-controller-yaml).
+
+| Parameter                                              | Parameter Type | Description                                                            | Required  |
+| ------------------------------------                   | -------------- | ---------------------------------------------------------------------- | --------- |
+| [namespaceConfig](#namespace-configuration-parameters) | Object         | The labels and annotations associated with the namespace.              | Mandatory |
```

Comment:
@gourishkb  - is the namespaceConfig mandatory? Is it only the user can set this if he wants to - meaning optional? 

cc @uma-kt

Reply:
confirmed with @gourishkb  that it's optional.

Reply:
done